go_modules/lib/dependabot/go_modules/package_manager.rb

```diff
@@ -0,0 +1,48 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+require "dependabot/ecosystem"
+require "dependabot/go_modules/version"
+require "dependabot/go_modules/requirement"
+
+module Dependabot
+  module GoModules
+    ECOSYSTEM = "go"
+    PACKAGE_MANAGER = "go_modules"
+    SUPPORTED_GO_VERSIONS = T.let([].freeze, T::Array[Dependabot::Version])
+
+    # When a version is going to be unsupported, it will be added here
+    DEPRECATED_GO_VERSIONS = T.let([].freeze, T::Array[Dependabot::Version])
+
+    class PackageManager < Dependabot::Ecosystem::VersionManager
+      extend T::Sig
+
+      sig do
+        params(
+          raw_version: String,
+          requirement: T.nilable(Requirement)
+        ).void
+      end
+      def initialize(raw_version, requirement = nil)
```

Comment:
If for go you don't have requirements you don't need to add it to your initialize method?

Reply:
I've cleaned this up.